server.js

```diff
@@ -46,24 +47,25 @@ const options = {
 const specs= swaggerJsDoc(options);
  
 // custom middleware
-app.use(logger);
+// app.use(logger);
 // this is for rendering the static files
 app.use(express.static(path.join(__dirname,'/public')));
 
 // third party middleware 
-const whiteList= ['http://127.0.0.1:5500','http://localhost:3000'];
-const corsOptions = {
-    origin:function(origin,callback){
-        if(whiteList.indexOf(origin) !== -1 || !origin){
-            callback(null, true)
-        }else{
-            callback(new Error("You are not allowed to access this apis"));
-        }
-    },
-    optionsSuccessStatus:200
-}
+// const whiteList= ['http://127.0.0.1:5500','http://localhost:3000'];
+// const corsOptions = {
```

Comment:
Can you remove these comment since you are not using them.

Reply:
i was studying about IP whitelisting, but after deployment i will add the real url

Reply:
Okay,